190114_tema/src/ro/sci/teo/ian14/Main.java

```diff
@@ -0,0 +1,110 @@
+package ro.sci.teo.ian14;
+
+import java.util.*;
+
+/**
+ * This is the main class, where all the program parts are put together.
+ *
+ * @author Teo
+ */
+public class Main {
+    public static void main(String[] args) {
+        //QUESTIONS:
+        //How do i make the product´s id unique? I considered it unique by default.
+        //How would you use an interface here?
+
+        OperateStore operateStore = new OperateStoreImpl();
+        //Here I populate the store.
+        List<Item> stock = new ArrayList<>();
```

Comment:
the Store should be responsible with the stock, not the main method. 
so the constructor of the store could do this initialization internally and keep the stock as an attribute, same for orders.